src/App.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useState } from "react";
+import UserHeader from "./components/header";
+import CharacterCard from "./components/characters";
+import { ContentStyle } from "./styled/style";
+import { characters } from "./data";
+
+type Character = {
+  id?: number;
+  image: string;
+  name: string;
+};
+
+function App() {
+  const [bestScore, setBestScore] = useState(0);
+  const [clicked, setClicked] = useState<number[]>([]);
+  const [score, setScore] = useState(0);
+
+  function shuffle(array: Character[]): void | Character[] {
```

Comment:
Why is this function returning a union of void & Character[] - `void | Character[]`

Union types are similar to `OR`, either this or that.
In our case however, the function is always going to return an array - so return type should just be `Character[]`

Reply:
I assumed an empty array as void here. Understood the mistake and rectifying it.